docs/quickstart-tenancy-and-apis.md

```diff
@@ -0,0 +1,314 @@
+# Quickstart: Tenancy and APIs
+
+## Prerequisites
+
+A running kcp server, as per [README.md](../README.md).
+
+## Set your KUBECONFIG
+
+To access a workspace, you need credentials and an Kubernetes API server URL for the workspace, both of which are stored
+in a `kubeconfig` file.
+
+The default context in the kcp-provided `admin.kubeconfig` gives access to the `root` workspace as the `kcp-admin` user.
+
+```shell
+$ export KUBECONFIG=.kcp/admin.kubeconfig
+$ kubectl config get-contexts
+CURRENT   NAME           CLUSTER   AUTHINFO      NAMESPACE
+          base           base      kcp-admin
+*         root           root      kcp-admin
+          system:admin   base      shard-admin
+```
+
+You can use API discovery to see what resources are available in this `root` workspace. We're here to explore the
+`tenancy.kcp.dev` and `apis.kcp.dev` resources.
+
+```shell
+$ kubectl api-resources
+NAME                              SHORTNAMES   APIVERSION                             NAMESPACED   KIND
+configmaps                        cm           v1                                     true         ConfigMap
+events                            ev           v1                                     true         Event
+...
+apibindings                                    apis.kcp.dev/v1alpha1                  false        APIBinding
+apiexports                                     apis.kcp.dev/v1alpha1                  false        APIExport
+...
+workspaces                        ws           tenancy.kcp.dev/v1beta1                false        Workspace
+```
+
+## Create and navigate some workspaces
+
+The `ws` plugin for `kubectl` makes it easy to switch your `kubeconfig` between workspaces, and to create new ones:
+
+```shell
+$ kubectl ws .
+Current workspace is "root".
+$ kubectl ws create foo --enter
+Workspace "foo" (type root:organization) created. Waiting for it to be ready...
+Workspace "foo" (type root:organization) is ready to use.
+Current workspace is "root:foo".
+$ kubectl ws create bar
+Workspace "bar" (type root:universal) created. Waiting for it to be ready...
+Workspace "bar" (type root:universal) is ready to use.
+$ kubectl get workspaces
+NAME   TYPE        PHASE   URL
+bar    universal   Ready   https://myhost:6443/clusters/root:foo:bar
+$ kubectl ws bar
+Current workspace is "root:foo:bar".
+$ kubectl ws root
```

Comment:
super-nit: can you do a `kubectl kcp ws ..` here, before going to root? Just to show it off.

Reply:
Done